src/MultivariateStats.jl

```diff
@@ -28,6 +29,8 @@ module MultivariateStats
     eigvecs,            # eignenvectors of the transformation
     loadings,           # model loadings
     var,                # model variance
+    tests,              # hypothesis tests
```

Comment:
```suggestion
```

(I don't think this is defined anywhere.)

Reply:
I removed it